source/publishing_data/12_translating_dataset/translating_dataset.rst

```diff
@@ -0,0 +1,88 @@
+Translating a dataset
+=====================
+
+For domains with more than one language activated in the front office, Opendatasoft allows the translation of specific elements of the datasets, as well as their metadata. It should help users understand a dataset as best as possible, even though the data itself is not in their preferred language.
+
+In the Opendatasoft platform, these translations are done in the Translations tab of the edition interface of each dataset.
+
+.. image:: images/translation_tab_interface.png
+
+Choosing elements for translation
+---------------------------------
+
+For each available language displayed in the Translations tab, it is possible to translate various elements that have been originally configured in 2 other configuration tabs of the dataset:
+
+- Information, which 4 following metadata can be translated: Title, Description, Keywords and Publisher.
```

Comment:
The "which" sounds weird to me, but I may be wrong. My brain would suggest "where the 4 following..." and "where fields' names and descriptions"

Reply:
Can't we translate all the metadata (custom/inspire ...) ?

Reply:
Not right now, we didn't want to spend time on it yet (including deciding whether it is useful or not; custom seems obvious, not sure about interop like dcat and inspire)

Reply:
Well, I'm not sure for the "where" instead of "which", but the sentence also sounded quite weird to me, so I modified the sentence to improve it. And I also added links to the documentations of every other configuration tab mentioned, and specified that the metadata that can be translated are standard metadata.